socialhome/streams/templates/streams/vue.html

```diff
@@ -0,0 +1,59 @@
+{% extends "streams/base.html" %}
+{% load staticfiles json_safe string_utils %}
+
+{% block content %}
+    {# Temporary hack; to be removed #}
+    <script>
+        window.templateData = {
+            contentList: [
+                {% for content in content_list %}
+                    {
+                        htmlSafe: {{ content.rendered|json_safe }},
+                        author: {
+                            guid: "{{ content.author.guid }}",
+                            name: "{% if content.author.name %}{{ content.author.name }}{% else %}{{ content.author.handle }}{% endif %}",
+                            handle: "{{ content.author.handle }}",
+                            imageUrlSmall: "{{ content.author.safer_image_url_small }}",
+                            absoluteUrl: "{{ content.author.get_absolute_url }}",
+                            homeUrl: "{{ content.author.home_url }}",
+                            isUserFollowingAuthor: {% if not content.author.id in request.user.profile.following_ids %}true{% else %}false{% endif %}
+                        },
+                        id: {{ content.id }},
+                        timestamp: "{{ formatted_timestamp }}",
+                        humanizedTimestamp: "{{ humanized_timestamp }}",
+                        edited: {{ content.edited|json_safe }},
+                        childrenCount: {{ content.children.count }},
+                        {% if content.author == request.user.profile %}
+                            isUserAuthor: true,
+                            updateUrl: "{% url " content:update" content.id %}",
+                            replyUrl: "{% url "content:reply" pk=content.id %}",
+                            deleteUrl: "{% url " content:delete" content.id %}",
+                        {% else %}
+                            isUserAuthor: false,
+                            updateUrl: "",
+                            replyUrl: "",
+                            deleteUrl: "",
+                        {% endif %}
+                    },
+                {% endfor -%}
+            ],
+            streamName: "{{ stream_name }}",
+            isUserAuthentificated: {% if request.user.is_authenticated %}true{% else %}false{% endif %},
+            showAuthorBar: {% if not stream_name|startswith:"profile_" %}true{% else %}false{% endif %},
+            {# Temporary hack; translation system to come #}
+            stampedContent: {
```

Comment:
To support other streams (now for example tags stream says "public" in the branch) you could move this object out into it's own for example `window.stampedContent`. And then in each of the Django templates that is rendered, set the correct texts.

The problem is some of the text is quite dynamic, for example profile stream. I wonder how that should be handled - it should really be vue components rendered `if streamName starts with "profile_"`. Maybe do that in follow up PR's and just leave this as is for now.